src/core/MRApp.js

```diff
@@ -193,6 +195,19 @@ export class MRApp extends MRElement {
      */
     init() {
         this.debug = this.getAttribute('debug') ?? false;
+
+        this.renderer = new THREE.WebGLRenderer({
+            antialias: true,
+            alpha: true, 
+            // There's issues in the timing to enable taking screenshots of threejs scenes unless you have direct access to the code.
+            // Using the preserveDrawingBuffer to ignore timing issues is the best approach instead. Though this has a performance hit,
+            // we're allowing it to be enabled by users when necessary.
+            //
+            // References:
+            // https://stackoverflow.com/questions/15558418/how-do-you-save-an-image-from-a-three-js-canvas
+            // https://stackoverflow.com/questions/30628064/how-to-toggle-preservedrawingbuffer-in-three-js
+            preserveDrawingBuffer: this.getAttribute('preserve-drawing-buffer') ?? false,
```

Comment:
Maybe we shorten it to `preserve-buffer`? Could just be my aversion to long attribute names.

Reply:
I agree with the reasoning; however, I'd rather not given this is us just exposing a threejs flag underneath so it's better practice for maintainability to keep them the same - if it was one we made ourselves though 100% shorter is better

Reply:
Fair point. maybe in the future as we expose more renderer options at the attribute level, we can fall back to an all encompassing `data-renderer` flag. something like.

`<mr-app data-renderer="antialias: true; alpha: true; preserveDrawingBuffer: true; ...">

rather than having one for each possible value.

Reply:
i like that idea - i'll open an issue for tracking and it'll probably stay on the backlog for a bit, but there's definitely other items in there we'd want to expose as well ~

Reply:
issue ref: #519